tests/client/test_client.py

```diff
@@ -1,9 +1,8 @@
 import asyncio
 import functools
 
-import pytest
```

Comment:
These lines look a lot like a failed isort to me. It _should_ know the known_first_party is `httpx`, but it's classifying it as a third party import.